src/components/ReferencesList/index.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+class ReferencesList extends Component {
+  render() {
+    const {references} = this.props;
+    return (
+      <div className="reference-list__wrapper">
+        <ul className="reference-list__list">
+          <h4>References</h4>
+          {references.map((item, index)=>{
+            return (
+              <li className="reference-list__item" key={index}>
+                <p className="reference-list__paragraph">
+                  {item.name}{', '}
+                  <span>
+                    <a href={item.contact}>
```

Comment:
Looks great @emma-martin! But I think maybe we need to use this example:
```
                   <a href="mailto:example@email.com">tu texto de anclaje</a>
```
or even put the mail address like a separate element? I mean like a separate link, after contact's role.
Something like that: 
```
               <div className="reference-list__paragraph">
                  <div className="reference-list__name">{item.name}</div>
                  <div className="reference-list__role">{item.role}</div>
                  <a className="reference-list__contact" href="mailto:{item.contact}">{item.contact}</a>
                </div>
```

Reply:
@maria-badanina good point!! Definitively keeping the mailto! What bothers me is how different it would look from the approximation offered by the client. Anyway, added the changes suggested and let's work around this in the afternoon :-)